SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/controller/MainController.java

```diff
@@ -359,40 +368,46 @@ public String shoppingCartCustomerSave(HttpServletRequest request, //
     @RequestMapping(value = { "/shoppingCartConfirmation" }, method = RequestMethod.GET)
     public String shoppingCartConfirmationReview(HttpServletRequest request, Model model) {
         CartInfo cartInfo = Utils.getCartInSession(request);
+        ShippingCost shippingCost = null;
  
-        // Cart have no products.
+        // No products in cart
```

Comment:
I liked the old wording better. Makes us sound like Russian scam artists.

Reply:
Haha! You're right, the original tone was much more credible :P